planning/behavior_path_planner/autoware_behavior_path_goal_planner_module/src/goal_planner_module.cpp

```diff
@@ -1363,17 +1363,25 @@ BehaviorModuleOutput GoalPlannerModule::planPullOver(const PullOverContextData &
 {
   universe_utils::ScopedTimeTrack st(__func__, *time_keeper_);
 
-  if (
-    path_decision_controller_.get_current_state().state !=
-    PathDecisionState::DecisionKind::DECIDED) {
-    return planPullOverAsCandidate(context_data);
+  const auto current_state = path_decision_controller_.get_current_state();
+  if (current_state.state != PathDecisionState::DecisionKind::DECIDED) {
+    const bool is_stable_safe = current_state.is_stable_safe;
+    // clang-format off
+    const std::string detail =
+      goal_candidates_.empty()                        ? "no goal candidate"              :
+      context_data.pull_over_path_candidates.empty()  ? "no path candidate"              :
+      !thread_safe_data_.foundPullOverPath()          ? "no static safe path"            :
+      !is_stable_safe                                 ? "unsafe against dynamic objects" :
+                                                        "not decided";
```

Comment:
What do you mean by "not decided"? It may be hard to understand for other people as well.

Reply:
There are several statuses and siince the transition to the `Decided` state occurs when several states are satisfied, I used `Not Decided` as it is to explain it in a few words.
But it may be hard for people who dones not read docduments.

![image](https://github.com/user-attachments/assets/34181c41-7a4b-47d7-8140-a29af4393d83)

Reply:
It is almost equal to “the goal is far away” or sometihing like that, so we change the text.

Reply:
fixed in 7c454ac99bbecb3dad483700e3e1b2b3fca27b43

use "goal is far"

Reply:
changed to "pull over start pose is far"

Reply:
It sounds too long. "too far goal" sounds much better to me, but I don't have a strong opinion on this

Reply:
changed to "too far goal" , not precise but simple and easy to understand for user